tests/plugins/toolsets/elasticsearch/test_elasticsearch_multi_instance.py

```
@@ -0,0 +1,503 @@
+
+class TestToolPathThreading:
+    @patch("holmes.plugins.toolsets.elasticsearch.elasticsearch.requests.request")
+    def test_tool_routes_to_requested_instance(self, mock_request):
+        mock_response = MagicMock()
+        mock_response.json.return_value = {"status": "green"}
+        mock_response.raise_for_status = MagicMock()
+        mock_request.return_value = mock_response
+
+        ts = _toolset_with(
+            username="elastic",
+            password="global-pw",
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to argument: "password"

(S106)